src/style.css

```
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+:root {
+  --color: #3d4e61;
+}
+
+html {
+  font-family: "Avenir", serif;
+  font-weight: 300;
+}
+
+.input-big {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.input {
+  font-family: inherit;
+  width: 100%;
+  color: var(--color);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.form {
+  margin: 20px 0;
+  position: relative;
+}
+
+.input-small {
+  height: 42px;
+  font-size: 14px;
+  padding-left: 33px;
+  background: white;
+  width: 100%;
+  color: var(--color);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.img {
+  position: absolute;
+  top: 25px;
```

it's bad way to center some absolute element by height; if input height change, you'll should find element and manually find top space to centring 

```suggestion
  top: 50%;
  transform: translateY(-50%)
```